kouncil-backend/pom.xml

```diff
@@ -141,6 +143,26 @@
             <version>1.1.9</version>
         </dependency>
 
+        <!--databases-->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <version>2.2.224</version>
```

Comment:
Add this version and also postgresql version to the <properties> as parameter variable